### ISSUE TITLE
Server crash In 1.19 with the update.

### ISSUE THREAD

**Issue by lilmnm-kamikaze-** [TrattMods/LiterallyChunkLoader#14]
Crafting the block works and placing it is fine. 
Right clicking it to open the interface the server crashes.

`[20:31:28] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.NoClassDefFoundError: net/minecraft/class_1074
        at net.literally.chunk.loader.entity.ChunkLoaderBlockEntity.method_5476(ChunkLoaderBlockEntity.java:28) ~[literally-chunk-loader-2.0.3-1.19.jar:?]
        at net.minecraft.class_3222.method_17355(class_3222.java:1047) ~[server-intermediary.jar:?]
        at net.literally.chunk.loader.block.ChunkLoaderBlock.method_9534(ChunkLoaderBlock.java:61) ~[literally-chunk-loader-2.0.3-1.19.jar:?]
        at net.minecraft.class_4970$class_4971.method_26174(class_4970.java:963) ~[server-intermediary.jar:?]
        at net.minecraft.class_3225.method_14262(class_3225.java:338) ~[server-intermediary.jar:?]
        at net.minecraft.class_3244.method_12046(class_3244.java:1115) ~[server-intermediary.jar:?]
        at net.minecraft.class_2885.method_12547(class_2885.java:34) ~[server-intermediary.jar:?]
        at net.minecraft.class_2885.method_11054(class_2885.java:8) ~[server-intermediary.jar:?]
        at net.minecraft.class_2600.method_11072(class_2600.java:22) ~[server-intermediary.jar:?]
        at net.minecraft.class_3738.run(class_3738.java:18) ~[server-intermediary.jar:?]
[8:33 PM]
at net.minecraft.class_1255.method_18859(class_1255.java:157) ~[server-intermediary.jar:?]
        at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_24306(MinecraftServer.java:776) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_18859(MinecraftServer.java:160) ~[server-intermediary.jar:?]
        at net.minecraft.class_1255.method_16075(class_1255.java:131) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_20415(MinecraftServer.java:758) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_16075(MinecraftServer.java:752) ~[server-intermediary.jar:?]
        at net.minecraft.class_1255.method_18857(class_1255.java:140) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_16208(MinecraftServer.java:737) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:669) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:258) ~[server-intermediary.jar:?]
        at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.ClassNotFoundException: net.minecraft.class_1074
        at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
        at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226) ~[fabric-loader-0.14.8.jar:?]
        at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:145) ~[fabric-loader-0.14.8.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
        ... 22 more`

Also to note its working fine in single player.

**Comment by overzealyss** [comment]
Same issue, anyone find a solve for this?

**Comment by Guimoute** [comment]
The mod works perfectly for me in the latest version:
- Minecraft 1.19.2
- Fabric Loader 0.14.9-1.19.2
- LiterallyChunkLoader 2.0.3-1.19

**Comment by DanHulton** [comment]
Exact same issue:
```
[01:14:04] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.NoClassDefFoundError: net/minecraft/class_1074
	at net.literally.chunk.loader.entity.ChunkLoaderBlockEntity.method_5476(ChunkLoaderBlockEntity.java:28) ~[literally-chunk-loader-2.0.3-1.19.jar:?]
	at net.minecraft.class_3222.method_17355(class_3222.java:1047) ~[server-intermediary.jar:?]
	at net.literally.chunk.loader.block.ChunkLoaderBlock.method_9534(ChunkLoaderBlock.java:61) ~[literally-chunk-loader-2.0.3-1.19.jar:?]
	at net.minecraft.class_4970$class_4971.method_26174(class_4970.java:964) ~[server-intermediary.jar:?]
	at net.minecraft.class_3225.method_14262(class_3225.java:338) ~[server-intermediary.jar:?]
	at net.minecraft.class_3244.method_12046(class_3244.java:1145) ~[server-intermediary.jar:?]
	at net.minecraft.class_2885.method_12547(class_2885.java:34) ~[server-intermediary.jar:?]
	at net.minecraft.class_2885.method_11054(class_2885.java:8) ~[server-intermediary.jar:?]
	at net.minecraft.class_2600.method_11072(class_2600.java:22) ~[server-intermediary.jar:?]
	at net.minecraft.class_3738.run(class_3738.java:18) ~[server-intermediary.jar:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:157) ~[server-intermediary.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_24306(MinecraftServer.java:776) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_18859(MinecraftServer.java:159) ~[server-intermediary.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:131) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_20415(MinecraftServer.java:758) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_16075(MinecraftServer.java:752) ~[server-intermediary.jar:?]
	at net.minecraft.class_1255.method_18857(class_1255.java:140) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_16208(MinecraftServer.java:737) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:669) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257) ~[server-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.ClassNotFoundException: net.minecraft.class_1074
	at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226) ~[fabric-loader-0.14.11.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:145) ~[fabric-loader-0.14.11.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	... 22 more
```
- Minecraft 1.19.2
- Fabric Loader 0.14.11-1.19.2
- LiterallyChunkLoader 2.03-1.19

Running it on a server as well - as soon as I right-click to access the interface, it crashes the server.

**Comment by tratteo** [comment]
Closed as duplicate of #15, check updates there
